### DISCUSSION TITLE
Remote Cache Docker Build

### DISCUSSION THREAD

**Discussion by StevenMatchett** [vercel/turborepo#1489] [Help]
I have stood up a standalone remote cache and everything works perfectly fine while running locally. 

When running in docker using (node:16-alpine) it is not using the remote cache.  Locally running the build I get `• Remote computation caching enabled (experimental)`, but do not see that in the docker output. I was curious if there was something that needs to be installed for turbo to work properly that my docker image doesn't have. I have tired to install git but that doesn't seem to fix the issue

**Comment by anthonyshew** [comment]
Closing this due to age and inactivity. If this is still a concern, would be great if we could get a reproduction on a fresh discussion.
